### ISSUE TITLE
Not working with vite and strict mode

### ISSUE THREAD

**Issue by utileetdulce** [dai-shi/react-hooks-worker#60]
Online Demo:
https://stackblitz.com/edit/vitejs-vite-dmru9t?file=src%2Fmain.jsx
When you remove the React.StrictMode in main.jsx the hook works as expected.

**Comment by dai-shi** [comment]
Yeah, this library was developed before strict effect was introduced. I don't think there's a reasonable solution with the current architecture.

Hmmm, that's annoying.
